skin/adminhtml/default/openmage/scss/_variables.scss

```diff
@@ -72,6 +72,70 @@ $font_2: Helvetica;
 $font_3: Arial;
 $font_4: sans-serif;
 
+//colors
+$color_mercury_approx: #e6e6e6;
+$color_mine_shaft_approx: #2f2f2f;
+$color_mango_tango_approx: #ea7601;
+$color_silver_chalice_approx: #aaa;
+$color_silver_approx: #c8c8c8;
+$white: #fff;
+$color_clementine_approx: #ed6502;
+$color_oregon_approx: #a04300;
+$color_yellow_orange_approx: #ffac47;
+$color_alabaster_approx: #f8f8f8;
+$black_25: rgba(0, 0, 0, 0.25);
+$black_50: rgba(0, 0, 0, 0.5);
+$color_pizazz_approx: #ff8c00;
+$color_flamingo_approx: #f16221;
+$black_15: rgba(0, 0, 0, 0.15);
+$black_75: rgba(0, 0, 0, 0.75);
+$color_gothic_10_approx: rgba(117, 146, 163, 0.1);
+$color_gothic_25_approx: rgba(117, 146, 163, 0.25);
+
+
+$hue: 205;
+/*
+                           205 = default,
```

Comment:
@ADDISON74 is right, why are these comments so far off?

Reply:
... they probably ran for fear of the edge on the left